### PR TITLE
Finish removing extra spaces from data and src folder

### PR DESCRIPTION
**Pull request by Cyrano7**

### CODE REVIEW

File: data/json/itemgroups/locations.json [hunk 1]
```diff
@@ -78,4 +78,3 @@
     ]
   }
 ]
-
```
Comment (Cyrano7) [inline data/json/itemgroups/locations.json]: So in my local version its showing that line 81 was deleted and line 82 is the new line 81. But here its showing that just line 81 was deleted and there is no line 82. Weird.

Comment (codemime) [inline data/json/itemgroups/locations.json]: Why do you also remove terminating newlines?

Comment (Cyrano7) [inline data/json/itemgroups/locations.json]: On my copy they are still there. There was an additional extra new line at the end of the file.

Comment (Cyrano7) [inline data/json/itemgroups/locations.json]: I'll sort it out just give me a sec.

**Comment by Cyrano7** [comment]
Might as well add the deletions from the src folder too.

**Pull request merged**